dss/storage/checkout/cache_flow.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+from dss.storage.hcablobstore import FileMetadata
+
+"""
+These functions assist with the caching process, the lifecycle policies are set to delete files within
+the checkout buckets;
+For AWS object tagging is used to mark files for deletion: TagSet=[{uncached:True}]
+For GCP object classes are used to indicate what is to be cached: STANDARD_STORAGE (MULTI-REGIONAL) are cached
+See MetaData Caching RFC for more information (Google Docs)
+"""
+
+
+def check_dss_bucket(dst_bucket: str):
```

Comment:
Can we think of a more descriptive name for this method, such as `should_cache_for_bucket`, or `is_dss_bucket`?